applications/FluidDynamicsApplication/tests/adjoint_vms_sensitivity_2d.py

```diff
@@ -0,0 +1,195 @@
+import os
+from KratosMultiphysics import *
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.FluidDynamicsApplication
+
+missing_applications_message = ["Missing required application(s):",]
+have_required_applications = True
+
+try:
+    import KratosMultiphysics.AdjointFluidApplication
+except ImportError:
+    have_required_applications = False
+    missing_applications_message.append("AdjointFluidApplication")
+
+try:
+    import KratosMultiphysics.HDF5Application as kh5
+except ImportError:
+    have_required_applications = False
+    missing_applications_message.append("HDF5Application")
+
+from fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+if have_required_applications:
+    from adjoint_fluid_analysis import AdjointFluidAnalysis
+
+class ControlledExecutionScope:
+    def __init__(self, scope):
+        self.currentPath = os.getcwd()
+        self.scope = scope
+
+    def __enter__(self):
+        os.chdir(self.scope)
+
+    def __exit__(self, type, value, traceback):
+        os.chdir(self.currentPath)
+
+@KratosUnittest.skipUnless(have_required_applications," ".join(missing_applications_message))
+class AdjointVMSSensitivity2D(KratosUnittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def _remove_file(self, file_path):
```

Comment:
It just a test but I would usually go for the naming suggestions of the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#classes-1)